src/Raven.Client/Documents/Subscriptions/SubscriptionWorker.cs

```diff
@@ -156,7 +157,7 @@ private Task Run(CancellationToken ct)
 
         private async Task<Stream> ConnectToServer(CancellationToken token)
         {
-            var command = new GetTcpInfoCommand("Subscription/" + _dbName, _dbName);
+            var command = new GetTcpInfoForRemoteTaskCommand("Subscription/" + _dbName, _dbName, _options?.SubscriptionName, verifyDatabase:true);
```

Comment:
Is there a compatibility concern here?
https://ravendb.net/docs/article-page/4.1/csharp/client-api/faq/backward-compatibility

So a 4.2 client will not work with a 4.1 or 4.0 server, so this is fine, according to the docs.

Reply:
It was the requirement that i'll move subscription to use the remote task, i could modify the database specific info/tcp to run the new logic and have a new entry point for the database/info/tcp that will handle this (this means duplicating code between the handlers), would that be better?

Reply:
When did we get the `/tcp/info/remote` endpoint?

Reply:
We got it for Pull replication not sure when that was added